theme.js

```diff
@@ -376,20 +406,41 @@ export default (variables = defaultThemeVariables) => ({
 
     lineHeight: formatLineHeight(variables.heading.fontSize),
     ...variables.heading,
+    fontFamily: resolveFontFamily(
+      variables.heading.fontFamily,
+      variables.heading.fontWeight,
+      variables.heading.fontStyle,
+    ),
+    fontWeight: Platform.OS === 'ios' ? variables.heading.fontWeight : 'normal',
```

Comment:
Would it make sense to extract this into the separate function since it's being used accross the theme?

Reply:
Yes it would, done and done. Great call!